features/metrics_versions_with_no_result.feature

```diff
@@ -0,0 +1,112 @@
+@api @metrics
+Feature: Versions with no result for a test
+
+  Retrieve the versions where there is no result for the test.
```

Comment:
If your description of the feature is going to be the same as the Cucumber feature title, don't write one. In this case, I would explain why it's useful to retrieve the versions where there are no results: it may allow you to identify tests that may not be running due to a technical issue.